README.md

```diff
@@ -7,6 +7,16 @@ Common code shared among
 [MySQL](https://github.com/juttle/juttle-mysql-adapter/)
 and [SQLite](https://github.com/juttle/juttle-sqlite-adapter/) adapters.
 
+#### List of optimized operations
+
+* any filter expression `read sql` (note: `read sql | filter ...` is not optimized)
```

Comment:
Our filter expressions in `read` generally support both globbing (covered by the tests you got) and regex. See the operator table here, what it says for ~ https://github.com/juttle/juttle/blob/master/docs/concepts/filtering.md
Now in practice a number of adapters don't support regex and will return an error if you try; you can always tack on an unoptimizable `filter` with the regex if you really need to. 
I just wanted to see where we stand with sql adapters.